go.sum

```diff
@@ -296,6 +296,10 @@ github.com/urfave/cli v1.22.1/go.mod h1:Gos4lmkARVdJ6EkW0WaNv/tZAAMe9V7XWyB60NtX
 github.com/urfave/cli/v2 v2.1.1/go.mod h1:SE9GqnLQmjVa0iPEY0f1w3ygNIYcIJ0OKPMoW2caLfQ=
 github.com/wasmerio/go-ext-wasm v0.3.1 h1:G95XP3fE2FszQSwIU+fHPBYzD0Csmd2ef33snQXNA5Q=
 github.com/wasmerio/go-ext-wasm v0.3.1/go.mod h1:VGyarTzasuS7k5KhSIGpM3tciSZlkP31Mp9VJTHMMeI=
+github.com/xeipuuv/gojsonpointer v0.0.0-20190905194746-02993c407bfb h1:zGWFAtiMcyryUHoUjUJX0/lt1H2+i2Ka2n+D3DImSNo=
+github.com/xeipuuv/gojsonpointer v0.0.0-20190905194746-02993c407bfb/go.mod h1:N2zxlSyiKSe5eX1tZViRH5QA0qijqEDrYZiPEAiq3wU=
+github.com/xeipuuv/gojsonreference v0.0.0-20180127040603-bd5ef7bd5415 h1:EzJWgHovont7NscjpAxXsDA8S8BMYve8Y5+7cuRE7R0=
+github.com/xeipuuv/gojsonreference v0.0.0-20180127040603-bd5ef7bd5415/go.mod h1:GwrjFmJcFw6At/Gs6z4yjiIwzuJ1/+UwLxMQDVQXShQ=
```

Comment:
I'm aware that I haven't been tracking the discussions that have happened about this before, so please bear with me -- I'm surprised about our vendoring choice here: We're manually vendoring the `github.com/xeipuuv/gojsonschema` code into `internal/`, and automatically vendor its dependencies via our ordinary means.

So... I'd think it would be more straightforward to only use the vendoring mechanisms we've been using for all other dependencies, i.e., not treat gojsonschema any special. This is assuming we don't need any changes to it -- maybe that's the reason and I'm missing the point here. _If that's the case_, let's drop a README into `internal/gojsonschem` to allow future readers to figure out why stuff was done the way it was done 😃

Reply:
(Or rather `github.com/johandorland/gojsonschema`, it seems.)

Reply:
Yes we have made a modification to the gojsonschema library. We need to use the result of the parser to traverse it and obtain a Rego type. The library was not originally designed for this use case, and the data structure that represents a parsed schema has private fields. We made these fields public. There are other users of this library who have asked for this modification over the years, but they don't seem inclined to accept a PR.

The other modifications have to do with linting and conforming to the code style.

We will add a README to this effect.

Reply:
@srenatus thank you for the review and finding nits. Very helpful! Re: `gojsonschema` as you guessed, the reason we manually vendored into `internal/` was because of changes we had to do to expose some of the schema and sub-schema fields relevant for our type-checking code in OPA. Will add a README to explain this for future users.

Also, the

Reply:
Done (commit 497cf35c844b333ec7e4fae74e375f22f035d083)